torch/onnx/_internal/exporter.py

```diff
@@ -1010,6 +1010,7 @@ def save(
         self,
         destination: Union[str, io.BufferedIOBase],
         *,
+        include_initializers: bool = True,
```

Comment:
how about passing the "no initializer" setting earlier such that graph building avoids copying initializer data?

Reply:
This is already possible using fake mode. The resulting fx/model proto will not have initializers from the beginning. Creating a second flag on export step might be confusing. e.g. what would happen with "include_initializers=True + fake mode? what if the user use "include_initializer=False" and no fake mode? how would that be different from fake mode from user (ux) point of view?

This PR abstracts however the graph was acquired - with or without initializer. It only gives the option for the user serializes in any way they want. Users could want no initializer during debug and with initializer when publishing the model, etc